app/src/main/java/com/example/taskmaster/MainActivity.java

```diff
@@ -382,15 +402,15 @@ public void onNothingSelected(AdapterView<?> parent) {
     public static PinpointManager getPinpointManager(final Context applicationContext) {
         if (pinpointManager == null) {
             final AWSConfiguration awsConfig = new AWSConfiguration(applicationContext);
-            AWSMobileClient.getInstance().initialize(applicationContext, awsConfig, new Callback<UserStateDetails>() {
+            AWSMobileClient.getInstance().initialize(applicationContext, awsConfig, new com.amazonaws.mobile.client.Callback<UserStateDetails>() {

```

Comment:
this code was broken because you used a different imported Callback class. I deleted it, typed "Callback" and picked the one that already had <UserStateDetails>, if you import an incorrect class for something (will happen plenty more times) your main clue will be that the method utilizing it ("initialize") will tell you the type is wrong. the class will also have red squiggles on the methods you try to override do not exist ("onResult" was not a method of the Callback class you chose)
this can happen when you copy paste and you already imported a class with the same name (you already were using a different "Callback" elsewhere)